sailpoint.sgnl.yaml

```diff
@@ -0,0 +1,1575 @@
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    attributes:
+      - name: name
+        externalId: name
+        type: String
+      - name: workflowName
+        externalId: workflowName
+        type: String
+        indexed: true
+        uniqueId: true
+
+  AlertActionResultNotifications:
+    parent: result
+    displayName: SailpointAlertActionResultNotifications
+    externalId: notifications
+    description: Entity representing the notifications associated with the Sailpoint Identity IQ Alert Action Result.
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    attributes:
+      - name: name
+        externalId: name
+        type: String
+      - name: displayName
+        externalId: displayName
+        type: String
+      - name: emailAddresses
+        externalId: emailAddresses
+        type: String
+        indexed: true
+        uniqueId: true
+
+  Applications:
+    displayName: SailpointApplication
+    externalId: Applications
+    description: Entity representing a Sailpoint Identity IQ Application.
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: id
+        externalId: id
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: name
+        externalId: name
+        type: String
+      - name: type
+        externalId: type
+        type: String
+      - name: features
+        externalId: features
+        type: String
+      - name: ownerValue
+        externalId: owner.value
+        type: String
+      - name: ownerRef
+        externalId: owner.["$ref"]
+        type: String
+      - name: ownerDisplayName
+        externalId: owner.displayName
+        type: String
+
+      # Meta
+      - name: resourceType
+        externalId: $.meta.resourceType
+        type: String
+      - name: created
+        externalId: $.meta.created
+        type: DateTime
+      - name: lastModified
+        externalId: $.meta.lastModified
+        type: DateTime
+      - name: version
+        externalId: $.meta.version
+        type: String
+      - name: location
+        externalId: $.meta.location
+        type: String
+
+  ApplicationSchemas:
+    parent: Applications
+    displayName: SailpointApplicationSchemas
+    externalId: applicationSchemas
+    description: Entity representing the schema associated with a Sailpoint Identity IQ Application.
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: value
+        externalId: value
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: type
+        externalId: type
+        type: String
+      - name: ref
+        externalId: $..["$ref"]
+        type: String
```

Comment:
What ref are you trying to access here? It looks like an application has a `ref` field nested under the `owner` and `applicationSchema` fields - and this should match both of them. If that's intentional I think this needs to be listed as a list, but this seems inconsistent with how this is handled elsewhere

https://developer.sailpoint.com/iiq/api/get-applications

Reply:
This refers to the `$ref` in the `applicationSchema`.

Reply:
Ahh that's my bad, I think I missed the `parent: Applications` and thought this was directly on the `Applications` entity